doc/getting-started.rst

```diff
@@ -0,0 +1,565 @@
+
+  pip install zc.buildout
+
+To use Buildout, you need to provide a Buildout configuration. Here is
+a minimal configuration::
+
+  [buildout]
+  parts =
+
+A minimal (and useless) Buildout configuration has a ``buildout`` section
+with a parts option.  If we run Buildout::
+
+  buildout
+
+Four directories are created:
+
+bin
+  A directory to hold executables.
+
+develop-eggs
+  A directory to hold develop egg links. More about these later.
+
+eggs
+  A directory that hold installed packages in egg [#egg]_ format.
+
+parts
+  A directory that provides a default location for installed parts.
+
+Buildout configuration files use an `INI syntax
+<https://en.wikipedia.org/wiki/INI_file>`_ [#configparser]_.
+Configuration is arranged in sections, beginning with section names in square
+brackets. Section options are names, followed by equal signs, followed
+by values.  Values may be continued over multiple lines as long as the
+continuation lines start with whitespace.
+
+Buildout is all about building stuff and the stuff to be built are
+specified using *parts*.  The parts to be built are listed in the
+``parts`` option.  For each part, there must be a section with the same
+name that names the software to build the part and optionally provides
+parameters to control how the part is built.
+
+Installing software
+===================
+
+In this tutorial, we're going to install a simple database server.
+The details of the server aren't important.  It just provides a useful
+example that illustrates a number of ways that Buildout can make
+things easier.
+
+We'll start by adding a part to install the server software.  We'll
+update our Buildout configuration to add a ``zeo`` part::
+
+  [buildout]
+  parts = zeo
+
+  [zeo]
+  recipe = zc.recipe.egg
+  eggs = ZEO
+
+We added the part name, ``zeo`` to the ``parts`` option in the
+``buildout`` section.  We also added a ``zeo`` section with two
+options:
+
+recipe
+  The standard ``recipe`` option names the software component that
+  will implement the part.  The value is a Python distribution
+  requirement, as would be used with ``pip``.  In this case, we've
+  specified `zc.recipe.egg
+  <https://pypi.python.org/pypi/zc.recipe.egg>`_ which is the name of
+  a Python project that provides a number of recipe implementations.
+
+eggs
+  A list of distribution requirements, one per
+  line. [#requirements-one-per-line]_ (The name of this option is
+  unfortunate, because the values are requirements, not egg names.)
+  Listed requirements are installed, along with their dependencies. In
+  addition, any scripts provided by the listed requirements (but not
+  their dependencies) are installed in the ``bin`` directory.
+
+If we run this [#gcc]_::
+
+  buildout
+
+Then a number of things will happen:
+
+- ``zc.recipe.egg`` will be downloaded and installed in your ``eggs``
+  directory.
+
+- ``ZEO`` and its dependencies will be downloaded and installed. (ZEO
+  is a small Python database server.)
```

Comment:
In this example, the directories should all be empty. Maybe I should say that, but I don't want to be caught in a lie. :) There may be eggs in the eggs directory because of an aborted upgrade, see https://github.com/buildout/buildout/issues/336.

Maybe I should show some partial listings later, when there should be things there. I'm not sure this would help. I'd like more opinions, but barring any I'll defer. :)

WRT windows, no-can-do.  If we decide ``bin`` listings are helpful I'd be happy to include a listing, but I'd need someone to contribute one.